lib/entities/core/ValetudoManualControlAction.js

```diff
@@ -0,0 +1,9 @@
+const ValetudoManualControlAction = Object.freeze({
+    FORWARD: "forward",
+    BACKWARDS: "backwards",
+    CLOCKWISE: "clockwise",
+    COUNTERCLOCKWISE: "counterclockwise",
+    STOP: "stop"
+});
+
+module.exports = ValetudoManualControlAction;
```

Comment:
This feels very ugly to me since all files in this directory contain `SerializableEntity`es. Let me know if there's a better place for it or a better way to do this.